ovos_persona_server/embeddings.py

```
@@ -0,0 +1,153 @@
+"""
+FastAPI router for OpenAI-compatible embeddings endpoints.
+
+This module provides an API endpoint for generating text embeddings
+using configured OVOS text embedding plugins.
+"""
+from contextlib import asynccontextmanager
+from typing import AsyncGenerator, List, Optional
+
+from fastapi import FastAPI, Depends, status, APIRouter, HTTPException
+from fastapi.responses import JSONResponse
+from ovos_plugin_manager.embeddings import load_text_embeddings_plugin, load_image_embeddings_plugin
```

Ruff (0.12.2)

`ovos_plugin_manager.embeddings.load_image_embeddings_plugin` imported but unused

Remove unused import: `ovos_plugin_manager.embeddings.load_image_embeddings_plugin`

(F401)